menas/src/main/resources/application.properties.template

```diff
@@ -60,6 +60,7 @@ za.co.absa.enceladus.menas.mongo.connection.database=menas
 za.co.absa.enceladus.migration.mongo.query.timeout.seconds=300
 
 za.co.absa.enceladus.menas.version=@project.version@
+za.co.absa.enceladus.menas.environment=localhost
```

Comment:
I am thinking how much it makes sense to have this in the properties file. 🤔

Reply:
It does not. And it does not for many more things. I hope this issue will solve it #510
I am not going against the flow for a small issue.